src/java.base/share/classes/java/net/doc-files/net-properties.html

```diff
@@ -214,6 +214,19 @@ <H2>Misc HTTP URL stream protocol handler properties</H2>
 	      property is defined, then its value will be used as the domain
 	      name.</P>
 	</OL>
+	<LI><P><B>{@systemProperty jdk.https.negotiate.cbt}</B> (default: &lt;never&gt;)<BR>
+	This controls the generation and sending of TLS channel binding tokens (CBT) when Kerberos 
+        or the Negotiate authentication scheme using Kerberos are employed over HTTPS with 
+        {@code HttpURLConnection}. There are three possible settings:</P>
```

Comment:
Should it be `{@code HttpsURLConnection}`?
(BTW - can we use {@code } here ? Would be worth checking the generated doc)

Reply:
Right HttpsURLConnection is better. {@code} works here.